docs/services/mira.md

```diff
@@ -50,9 +51,11 @@ Mira supports the following operation modes:
 |[Swarm](#swarm-mode)                   | Discovers Qlik Associative Engine instances in a Docker Swarm environment.            |
 |[Kubernetes](#kubernetes-mode)         | Discovers Qlik Associative Engine instances in a Kubernetes environment.              |
 |[DNS](#dns-mode)                       | Discovers Qlik Associative Engine instances using DNS service look-ups.               |
-|[Local](#local-mode)                   | Discovers Qlik Associative Engine instances running on the local Docker Engine.        |
+|[Local](#local-mode)                   | Discovers Qlik Associative Engine instances running on the local Docker Engine.       |
 
 To set the operation mode, define the environment variable `MIRA_MODE` on the Mira container.
+In `Local` or `Swarm` mode you will also have to set the running user to `root` in your `docker-compose.yml` file.
```

Comment:
Usually when you do backticks, it refers to the actual code value you use (e.g. `local` or `swarm`)